### PR TITLE
Merge `equalizer` & `equalizer_exchange` into `equalizer`

### PR DESCRIPTION
**Pull request by hildobby**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by hildobby** [comment]
@jeff-dude I've seen this error pop up more and more lately in PRs modifying existing spells:`TrinoExternalError(type=EXTERNAL, name=DELTA_LAKE_INVALID_SCHEMA, message="Metadata not found in transaction log for test_data.dex_trades_seed", query_id=20231107_224306_01342_s7shm)`

Any solutions to fix it other than appending a line to schema files so they are factored into CI tests (that's my current non-ideal solution)? I don't remember this being an issue lately, @0xRobin also mentioned seeing this increasingly frequently this weekend

**Comment by jeff-dude** [comment]
> @jeff-dude I've seen this error pop up more and more lately in PRs modifying existing spells:`TrinoExternalError(type=EXTERNAL, name=DELTA_LAKE_INVALID_SCHEMA, message="Metadata not found in transaction log for test_data.dex_trades_seed", query_id=20231107_224306_01342_s7shm)`
> 
> Any solutions to fix it other than appending a line to schema files so they are factored into CI tests (that's my current non-ideal solution)? I don't remember this being an issue lately, @0xRobin also mentioned seeing this increasingly frequently this weekend

it doesn't seem spellbook specific, as the error occurs when querying the seed on dune too.
image

i forget what this is / why it's happening. i'll need to check with the engine team to see.

File: models/equalizer/fantom/equalizer_fantom_pools.sql [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
                                 "project",
-                                "equalizer_exchange",
+                                "equalizer",
                                 \'["Henrystats"]\') }}'
     )
 }}
 
 SELECT 'fantom' AS blockchain
-, 'equalizer_exchange' AS project
+, 'equalizer' AS project
```
Comment (jeff-dude) [inline models/equalizer/fantom/equalizer_fantom_pools.sql:16]: can you also add `schema` property in config block of all these models?
then remove schema assignment for equalizer in dbt_project file?

it'll be good to get head start on that for this dex as we do the same for others moving forward

**Comment by 0xRobin** [comment]
> i forget what this is / why it's happening. i'll need to check with the engine team to see.

If I remember correctly this happens when you query a temporary CI table that has already been cleaned up.

**Comment by jeff-dude** [comment]
> If I remember correctly this happens when you query a temporary CI table that has already been cleaned up.

even the seeds we build in prod now give that error when querying them. i believe that's the new issue. we see the same error on dune app and CI tests.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**